scripts/lisk.sh

```diff
@@ -19,40 +13,44 @@ fi
 # shellcheck disable=SC1090
 . "$(pwd)/env.sh"
 
+# Declare static config for redis
+REDIS_CONFIG="$(pwd)/etc/redis.conf"
+REDIS_BIN="$(pwd)/bin/redis-server"
+REDIS_CLI="$(pwd)/bin/redis-cli"
+REDIS_PID="$(pwd)/redis/redis_6380.pid"
 
-PM2_CONFIG="$(pwd)/etc/pm2-lisk.json"
-PM2_APP="$(grep "name" "$PM2_CONFIG" | cut -d'"' -f4)" >> /dev/null
-LISK_CONFIG="$(grep "config" "$PM2_CONFIG" | cut -d'"' -f4 | cut -d' ' -f2)" >> /dev/null
-LISK_LOGS="$(grep "logFileName" "$LISK_CONFIG" | cut -f 4 -d'"')"
+# Declare static config for postgresql
+DB_DATA="$(pwd)/blockchain.db"
+DB_LOG_FILE="$(pwd)/pgsql.log"
+DB_SNAPSHOT="blockchain.db.gz"
+DB_DOWNLOAD=Y
+
+PM2_MAIN_CONFIG="$(pwd)/etc/pm2-lisk-main.json"
+PM2_TEST_CONFIG="$(pwd)/etc/pm2-lisk-test.json"
+PM2_DEV_CONFIG="$(pwd)/etc/pm2-lisk-dev.json"
 
 LOGS_DIR="$(pwd)/logs"
 
-# Allocates variables for use later, reusable for changing pm2 config.
 config() {
-DB_NAME="$(grep "database" "$LISK_CONFIG" | cut -f 4 -d '"')"
-DB_PORT="$(grep "port" "$LISK_CONFIG" -m2 | tail -n1 |cut -f 1 -d ',' | cut -f 2 -d ':')"
-DB_USER="$USER"
-DB_PASS="password"
-DB_DATA="$(pwd)/pgsql/data"
-DB_LOG_FILE="$LOGS_DIR/pgsql.log"
-DB_SNAPSHOT="blockchain.db.gz"
-DB_DOWNLOAD=Y
-
-REDIS_CONFIG="$(pwd)/etc/redis.conf"
-REDIS_BIN="$(pwd)/bin/redis-server"
-REDIS_CLI="$(pwd)/bin/redis-cli"
-REDIS_ENABLED="$(grep "cacheEnabled" "$LISK_CONFIG" | cut -f 2 -d ':' |  sed 's: ::g' | cut -f 1 -d ',')"
-REDIS_PORT="$(grep "port" "$LISK_CONFIG" -m3 | sed -n 3p | cut -f 2 -d':' | sed 's: ::g' | cut -f 1 -d ',')"
-REDIS_PASSWORD="$(grep "password" "$LISK_CONFIG" -m2 | sed -n 2p | cut -f 2 -d ":" | cut -f 1 -d ',' | sed 's: ::g')"
-REDIS_PID="$(pwd)/redis/redis_6380.pid"
+	# Initalize application specific Postgresql variables
+	DB_NAME="$(jq .db.database -r "$LISK_CONFIG")"
+	DB_PORT="$(jq .db.port -r "$LISK_CONFIG")"
+	DB_USER="$USER"
+	DB_PASS="$(jq .db.password -r "$LISK_CONFIG")"
+
+	# Initialize application specific Redis variables
+	REDIS_ENABLED="$(jq .cacheEnabled -r "$LISK_CONFIG")"
+	REDIS_PORT="$(jq .redis.port -r "$LISK_CONFIG")"
+	REDIS_PASSWORD="$(jq .redis.port -r "$LISK_CONFIG")"
```

Comment:
Looks like a typo; should be:

     REDIS_PASSWORD="$(jq .redis.password -r "$LISK_CONFIG")"

Reply:
Good catch. Corrected this one as well.